### PR TITLE
Notifications for user/org/community messages

### PR DESCRIPTION
**Pull request by alstojanovic**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: service/src/templates/communication.community.leads.message.recipient.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+/* eslint-disable quotes */
+module.exports = () => ({
+  name: 'communication.community.leads.message.recipient',
+  title: '{{messageSender.displayName}} sent a message to your community!',
+  version: 1,
+  channels: {
+    email: {
+      from: '{{emailFrom}}',
+      to: '{{recipient.email}}',
+      replyTo: '{{messageSender.email}}',
```
Comment (hero101) [inline service/src/templates/communication.community.leads.message.recipient.js:12]: This is exposing the sender's email - is it discussed with Product?

**Review by hero101 — CHANGES_REQUESTED** [review]

File: service/src/templates/communication.community.leads.message.recipient.js (hunk 1 above)
Comment (alstojanovic) [reply service/src/templates/communication.community.leads.message.recipient.js:12]: image
Yes, this was intentional.

File: lib/src/dto/communication.community.leads.message.payload.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { JourneyBaseEventPayload } from "./journey.base.event.payload";
+
+export interface CommunicationCommunityLeadsMessageEventPayload extends JourneyBaseEventPayload {
+    message: string;
+    messageSender: {
```
[the remark this answers is not in the document]
Comment (alstojanovic) [reply lib/src/dto/communication.community.leads.message.payload.ts:5]: I was thinking about that. Originally it was only messageSenderId, I will change it back.

[the remark this answers is not in the document]
Comment (hero101) [reply lib/src/dto/communication.community.leads.message.payload.ts:5]: Well maybe you can include the name or something - I saw that you did it somewhere else

**Review by hero101 — APPROVED** [review]

**Pull request merged**